pkg/umorse/Kconfig

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+config PACKAGE_UMORSE
+    bool "Morse code encoder"
```

Maybe we should mention the package name in the prompt? Something like:
```suggestion
    bool "uMorse - Morse code encoder"
```